src/components/20-molecules/file-upload/index.js

```diff
@@ -34,6 +34,14 @@ const IMAGE_FILE_TYPES = 'image/jpg, image/jpeg, image/png';
 
 export const getBytesFromKilobyte = kilobyte => 1024 * kilobyte;
 
+const findFileById = (files, file, indexOnly) => {
+  for (let i = 0, n = files.length; i < n; i++) {
```

Comment:
Are there any advantages over `for (let i = 0; i < files.length; i++) {`?

Reply:
Yes, you don't evaluate the length of an array in every iteration!